pandora-client-web/src/components/chatroom/chatInput.tsx

```diff
@@ -467,7 +468,7 @@ function TextAreaImpl({ messagesDiv, scrollMessagesView }: {
 
 	useEffect(() => () => inputEnd(), [inputEnd]);
 
-	return <textarea ref={ ref } onKeyDown={ onKeyDown } onChange={ onChange } onBlur={ inputEnd } defaultValue={ InputRestore.value.input } />;
+	return <textarea ref={ useTextFormatting(ref) } onKeyDown={ onKeyDown } onChange={ onChange } onBlur={ inputEnd } defaultValue={ InputRestore.value.input } />;
```

Comment:
Little nitpick: You use it once directly and once indirectly in the files where I placed this comment.